blueocean-core-js/src/js/index.js

```diff
@@ -9,6 +9,18 @@ import { RunApi } from './rest/RunApi';
 
 import { SseBus } from './sse/SseBus';
 import { ToastService } from './ToastService';
+import {
```

Comment:
I think my IDE is being sneaky and reformatting this file. can probably just revert.